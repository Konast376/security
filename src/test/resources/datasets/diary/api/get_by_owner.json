{
  "diary": [
    {
      "id": "00000000-0000-0000-0000-000000000000",
      "owner": "user",
      "title": "1",
      "record": "first",
      "record_date": "2019-05-20 21:15:30.0"
    },
    {
      "id": "00000000-0000-0000-0000-000000000001",
      "owner": "user2",
      "title": "2",
      "record": "first",
      "record_date": "2019-05-20 21:15:30.0"
    },
    {
      "id": "00000000-0000-0000-0000-000000000002",
      "owner": "user3",
      "title": "3",
      "record": "first",
      "record_date": "2019-05-20 21:15:30.0"
    },
    {
      "id": "00000000-0000-0000-0000-000000000003",
      "owner": "user2",
      "title": "4",
      "record": "first",
      "record_date": "2019-05-20 21:15:30.0"
    }
  ]
}