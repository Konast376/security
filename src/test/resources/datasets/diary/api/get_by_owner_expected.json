{
  "diary": [
    {
      "id": "00000000-0000-0000-0000-000000000000",
      "owner": "user1",
      "title": "1",
      "record": "first",
      "record_date": "2019-05-20 21:15:30.0"
    }
  ]
}